### PR TITLE
add ecs json config

### PR DESCRIPTION
**Pull request by Mirak624**
ECS JSON config for Netsil collectors
Please confirm any resource limits and let me know if they should be adjusted.
No Health Check (should not be necessary)

### CODE REVIEW

File: ecs/netsil-collectors.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+               "name":"SD_BACKEND",
+               "value":"docker"
+            },
+            {
+               "name":"DEPLOY_ENV",
+               "value":"docker"
+            },
+            {
+               "name":"NETSIL_ORGANIZATION_ID",
+               "value":${organizationId}
```
Comment (sasounda) [inline ecs/netsil-collectors.json]: https://trello.com/c/W8BZq8T9/2702-template-ecs-specs-in-the-docs for making this embedded

File: ecs/netsil-collectors.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            },
+            {
+               "name":"SAMPLINGRATE",
+               "value":"100"
+            },
+            {
+               "name":"NETSIL_SP_HOST",
+               "value":${your_netsil_ip}
```
Comment (sasounda) [inline ecs/netsil-collectors.json]: same here

File: ecs/netsil-collectors.json [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+         ],
+         "workingDirectory":null,
+         "dockerSecurityOptions":null,
+         "memory":1024,
+         "memoryReservation":null,
+         "volumesFrom":[
+
+         ],
+         "image":"netsil/collectors:latest",
```
Comment (sasounda) [inline ecs/netsil-collectors.json]: @tanveergill do we want to force pull image ?

**Comment by tanveergill** [comment]
just set the mem limits based on k8s spec

**Pull request merged**